### PR TITLE
Reference inject_credentials at its new home

### PR DESCRIPTION
**Pull request by chrismeyersfsu**
##### SUMMARY
Reference inject_credentials at its new home

##### ISSUE TYPE

 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME

 - API

##### AWX VERSION

```
devel
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by AlanCoding** [comment]
So what needs to be merged before this is functional?

File: awx/main/models/credential.py [hunk 1]
```diff
@@ -688,7 +556,7 @@ def __init__(self, namespace, **kwargs):
         for k in ('inputs', 'injectors'):
             if k not in kwargs:
                 kwargs[k] = {}
-        kwargs.setdefault('custom_injectors', None)
+        # kwargs.setdefault('custom_injectors', None)
```
Comment (AlanCoding) [inline awx/main/models/credential.py]:
```suggestion
```

**Comment by chrismeyersfsu** [comment]
The tests moved here https://github.com/ansible/awx_plugins.interfaces/pull/19/files#diff-a3de1509190468679e3a2e296e2a882dc26d472a2ab837e425d2f37a4bdb2a5cR87

**Review by AlanCoding — APPROVED** [review]

**Comment by AlanCoding** [comment]
In https://github.com/ansible/awx/pull/16092, I'm trying to go over test failures, and saw this fail:

```
  File "/awx_devel/awx/main/tests/unit/test_tasks.py", line 1030, in test_custom_environment_injectors_with_extra_vars
    assert hasattr(extra_vars["api_token"], '__UNSAFE__')
AssertionError: assert False
 +  where False = hasattr('ABC123', '__UNSAFE__')
```

And that was removed here, in this PR. Ok fine, it was moved to https://github.com/ansible/awx_plugins.interfaces

However, try as I might, I can not find a test exercising this "__UNSAFE__" behavior.

https://github.com/search?q=repo%3Aansible%2Fawx_plugins.interfaces%20__UNSAFE__&type=code

Should I not worry about this? I felt sure that marking yaml vars as unsafe was still intended. For the git diff, the right path still seems to be to delete the test but I just want to make sure that something didn't fall through the cracks.

**Comment by chrismeyersfsu** [comment]
Looks like that was a big miss. We are now calling `yaml.safe_dump` [here](https://github.com/ansible/awx_plugins.interfaces/blob/67eba3dcd9148c1b01ebc07e1d2dcb3a4fb6708d/src/awx_plugins/interfaces/_temporary_private_inject_api.py#L152) which is NOT the same as `awx.main.utils.safe_yaml.safe_dump`

**Pull request merged**